### ISSUE TITLE
"Mutex.h: No such file or directory"

### ISSUE THREAD

**Issue by Meganium97** [DaeFennek/libpega#12]
Got this error while running the make command with devkitpro installed. I'm running ubuntu 21.04. Any hep I can get would be appreciated.

**Comment by xerpi** [comment]
Yeah, there's a mix of upper case and lower case filenames which is a mess. This is what happens when you develop in a case-insensitive filesystem...

**Comment by ColorVoid** [comment]
hi! could anyone please help me with this error im having

**Comment by kperdlich** [comment]
I might clean that up, but yeah I created this project on a windows maschine. So there can be problems.
